app/jobs/sync/zendesk/ticket_comments_to_notes.rb

```diff
@@ -0,0 +1,50 @@
+class Sync::Zendesk::TicketCommentsToNotes
+  include Sidekiq::Worker
+
+  # https://developer.zendesk.com/rest_api/docs/support/ticket_comments#list-comments
+  def perform(zendesk_ticket_id, page_number = 0)
+    require 'zendesk_api'
+    client = ZendeskAPI::Client.new do |config|
+      # Mandatory:
+      config.url = "#{Rails.application.secrets[:zendesk_url]}/api/v2" # e.g. https://mydesk.zendesk.com/api/v2
+
+      # Basic / Token Authentication
+      config.username = Rails.application.secrets[:zendesk_username]
+
+      # Choose one of the following depending on your authentication choice
+      # config.token = "your zendesk token"
+      # config.password = "your zendesk password"
+
+      # OAuth Authentication
+      config.access_token = Rails.application.secrets[:zendesk_access_token]
+
+      # Optional:
+
+      # Retry uses middleware to notify the user
+      # when hitting the rate limit, sleep automatically,
+      # then retry the request.
+      config.retry = true
```

Comment:
Should we consider logging when this happens

Reply:
Retries are noise most of the time. I'll log entry and exit though